tests_integration/conftest.py

```diff
@@ -105,3 +107,27 @@ def invoke_integration_without_format(self, *args, **kwargs) -> CommandResult:
 @pytest.fixture
 def runner(test_snowcli_config_provider):
     return SnowCLIRunner(app, test_snowcli_config_provider)
+
+
+@pytest.fixture
+def project_file(temporary_working_directory, test_root_path):
```

Comment:
We will have to merge it with Adam's version from #437.

And I'd like to keep the names used in Adam's version.

Reply:
I removed it as it's no longer neccessary because we use result of `init` command